src/components/composer/content_editable_helper.ts

```diff
@@ -274,3 +276,9 @@ function compareContentToSpanElement(content: HtmlContent, node: HTMLElement): b
   const sameContent = node.innerText === content.value;
   return sameColor && sameClass && sameContent;
 }
+
+function isEmptyParagraph(node: Node) {
+  return ["<br>", "<span><br></span>"].includes(
+    (node as HTMLElement).innerHTML.replaceAll(/\s(class|style)="[^"]*"/g, "")
```

Comment:
It doesn't behave as expected if I type ` class="sdf"` in a cell (plain text).

Also, it doesn't seem be problematic in this specific case, but as always, playing with `innerHTML` is dangerous. It doesn't trigger the security check in odoo, but it should if it was stricter

Any easy other way to achieve the same thing ?

Reply:
the good solution would be to use innerText all the way but requires some polyfill for the tests, I'm not sure we will reach a "good" point where we don't end up testing the polyfill in the existing tests though

Reply:
@LucasLefevre I added a wip commit with an alternative implementation using actual node comparison. I did not notice any performance issue with it and the initial problem seems fixed as well :)